mvpa2/algorithms/hyperalignment.py

```diff
@@ -198,7 +198,7 @@ def train(self, datasets):
             ref_ds = np.argmax(nfeatures)
         else:
             ref_ds = params.ref_ds
-            if ref_ds < 0 and ref_ds >= ndatasets:
+            if ref_ds >= ndatasets:
```

Comment:
I like that it gives an informative error message for a too large value of ref_ds, but what about giving a non-integer or negative value?